# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/jimmy/zgv/gcc-3.3.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./src/bdf2h.c.orig	2000-06-01 17:44:48.000000000 +0300
+++ ./src/bdf2h.c	2003-10-19 09:26:48.000000000 +0300
@@ -45,14 +45,14 @@
  */
 printf("#ifndef DEFINED_STRUCT_FONTINFO_TAG\n");
 printf("#define DEFINED_STRUCT_FONTINFO_TAG 1\n");
-printf("\
-struct fontinfo_tag
-  {
-  signed char *data;
-  int table[96];
-  int yofs,fh,oy;
-  };
-");
+printf(
+"struct fontinfo_tag\n"
+"  {\n"
+"  signed char *data;\n"
+"  int table[96];\n"
+"  int yofs,fh,oy;\n"
+"  };\n"
+);
 printf("#endif\n\n");
 
 /* generate font data */
